### ISSUE TITLE
Blank screenshots are captured when restartBrowserBetweenTests: true

### ISSUE THREAD

**Issue by sagarm-optimus** [etxebe/protractor-html-reporter#1]
I am using this package for taking screenshots and reports (whatever is given in the 'How to Use' section)

I am using jasmine framework. When restartBrowserBetweenTests: true is set in protractor's config file, browser is closed after each test case and started again with the next test case. The issue is that with this condition, failed cases' screenshots being captured are blank. 

When I set it to false, screenshots are captured correctly.

This setting restartBrowserBetweenTests: true itself has issues and as work around I am using protractor.browser.takeScreenshot instead of just browser.takeScreenshot. 

To summarize, when restartBrowserBetweenTests is:
1) False: everything works fine, i.e. with both protractor.browser.takeScreenshot and protractor.browser
2) True: nothing works fine

**Comment by etxebe** [comment]
I have never used restartBrowserBetweenTests option. Look at this issue: http://stackoverflow.com/questions/39504724/protractor-restartbrowserbetweentests-true-restart-browser-but-dont-run-tes.
Maybe you should use BeforeEach() and AfterEach() instead of?

**Comment by etxebe** [comment]
Besides, when I try to run my tests with this option I get errors (even if I disable taking screenshots): "  - Failed: This driver instance does not have a valid session ID (did you call WebDriver.quit()?) and may no longer be used.
Failed: Error while waiting for Protractor to sync with the page: "window.angular is undefined.  This could be either because this is a non-angular page or because your test involves client-side navigation, which
 can interfere with Protractor's bootstrapping.  See http://git.io/v4gXM for details""
